kata_1.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import "fmt"
+
+// struct -> table with only one column, each can use different types
+type Stack struct {
+	Stack []int
```

Comment:
I would suggest naming it `elements` or `items`, because your struct is the "stack", which contains elements. Its not a stack which contains a stack.